api/LNSocketAPI.ts

```diff
@@ -6,10 +6,20 @@ import {
   ListOffers,
   OfferDecode,
 } from "../model/CoreLN";
-import { MetricsOne } from "../model/Metrics";
 import axios from "axios";
 import { OfferInfo } from "../model/Offer";
 import { inject, singleton } from "tsyringe";
+import {
+  ApolloClient,
+  InMemoryCache,
+  ApolloProvider,
+  gql,
+} from "@apollo/client";
+
+const client = new ApolloClient({
+  uri: "https://api.lnmetrics.info/query",
+  cache: new InMemoryCache(),
+});
```

Comment:
```suggestion
const client = 
```

this need to be declared inside the class 

```
export default class LNSocketAPI implements AppAPI {
  private address: string;
  private nodeID: string;
  private rune: string;
  private lambda: string;
  private ApolloClient gqlClient;

  constructor(
    @inject("lambda") lambda: string,
    @inject("nodeID") nodeId: string,
    @inject("address") adderss: string,
    @inject("rune") rune: string,
   @inject("lnmetrics_url") url: string,
  ) {
    this.nodeID = nodeId;
    this.address = adderss;
    this.rune = rune;
    this.lambda = lambda;
    this.gqlClient = new ApolloClient({
  uri: url,
  cache: new InMemoryCache(),
});
  }